eudi_wallet_python/tools/qr_code.py

```diff
@@ -0,0 +1,36 @@
+import zlib
+import qrcode
+from PIL import Image
+
+class QRCode():
+    def __init__(self, data: str, base_size: dict = {"w": 100, "h": 100}, color: str = '#2B4375', logo_path: str = None, use_zlib: bool = True) -> None:
```

Comment:
please move these default parameters values base_size and color in the backend configuration

Reply:
Resolved in ec8d4a3cc71a2453ace1bbe14dd6aa298314e2be